op-node/node/node.go

```diff
@@ -304,6 +308,19 @@ func (n *OpNode) initL1BeaconAPI(ctx context.Context, cfg *Config) error {
 	cl := sources.NewL1BeaconClient(httpClient)
 	n.beacon = cl
 
+	if cfg.Rollup.EcotoneTime != nil {
+		vctx, cancel := context.WithTimeout(ctx, time.Second*10)
+		defer cancel()
+
+		v, err := cl.GetVersion(vctx)
+
+		if err != nil {
+			return fmt.Errorf("failed to check beacon api version: %w", err)
```

Comment:
Entirely aborting the start of the op-node without beacon-api, before the Ecotone hardfork activates, seems a bit harsh for users who may still be making infra changes. And if the beacon-chain has a problem, then ideally we are still able to operate the op-node in case of emergency. It's otherwise good to signal though, as it is a strict requirement for DA after Ecotone.

Can we add a `l1.beacon.ignore` CLI boolean flag? And if it's true, turn the error here in a warn log and return nil?

Reply:
Will do 👍